api/src/api/users/user_routes.py

```diff
@@ -282,7 +282,7 @@ def user_save_search(
         },
     )
 
-    return response.ApiResponse(message="Success")
+    return response.ApiResponse(message="Success", data=saved_search)
```

Comment:
@chouinar would appreciate a look at this, as I imagine it will break some tests, and there may be a better way of going about getting the id down in the payload

Reply:
Could we spin this change off into a separate PR? There's a lot going on here, and I want to make sure the API change gets the right attention / is implemented for what you are actually looking for

Reply:
sure

Reply:
https://github.com/HHS/simpler-grants-gov/pull/4196